apps/07jquery-tic-tac-toe/script.js

```diff
@@ -1,5 +1,44 @@
 'use strict';
 
-$(document).on('ready', function() {
-  // Put app logic in here
-});
+$(document).on('ready', function () {
+
+    var winConditionX = 0;
+    var winConditionO = 0;
+    var winArray = [7, 56, 448, 73, 146, 292, 273, 84];
+    var playerTurn = "X";
+
+    $('[data-cell]').click(function () {
+        var $this = $(this);
+        if ($this.text()==="X" || $this.text() ==="O"){
```

Comment:
You could use do `if ($this.text()) { .... }` to check if a playerTurn has been made there.